### ISSUE TITLE
Fix for OF scraper ignoring input and not downloading content

### ISSUE THREAD

**Issue by Throwaway115** [DIGITALCRIMINALS/OnlyFans#571]
I think ive fixed mine!
I set messages and paid content to false and it seems to be downloading content for me.
its not everything but at least its something

![image](https://user-images.githubusercontent.com/77456400/182955688-3cddd219-2059-4713-9c1d-91239b088ab2.png)

Edit: Interestingly it still downloads paid content and message even though its set to false

_Originally posted by @Throwaway115 in https://github.com/DIGITALCRIMINALS/OnlyFans/issues/557#issuecomment-1205785793_

**Comment by blueoceanwaves** [comment]
Thank you, I just tried it with setting messages to false and it seems to be working.

**Comment by Throwaway115** [comment]
> 

Glad to hear it helped!

**Comment by interbiznw** [comment]
I select one model, it looks like its downloading, then says scrape completed ( nothing is downloaded) then it proceeds to go on to "scraping paid content" then starting scaping another model, which I didn't even tell it to do?? Anyone have this issue?

**Comment by Leboxis** [comment]
I have the same :/

**Comment by Leboxis** [comment]
It's working for me when I turn messages to false

**Comment by MonkeyKingViper** [comment]
Maybe this is happening for others, maybe not.

I followed the advice of the original post because it seemed I was having trouble with a certain creator. I noticed I was missing a lot of videos including recent ones.

I turned messages and paid to false and started the scraper while watching the video folder. The total jumped from 16 to 22 and included the ones that were missing. And then suddenly they disappeared in front of me and I was back to 16. So I started the scraper again and cancelled out right after they appeared so I could move those files to a separate folder.

In the original I tried to go back to the videos folder but instead of under posts, I accidentally did messages. And I saw one of the videos I knew I was missing. I looked at the file name and realized they had the same name. I searched all the videos I was missing and found them in that folder.

So it wasn't that I was ever missing them, but that they exited somewhere else and the dupe checker or something was keeping it from downloading a copy. When I was seeing them reappear and disappear, they were being reorganized in the folders.

Is it possible that the "not scraping" is just you not realizing that you have that same content in another folder?
